pkg/processor/build/runtimeconfig/runtimeconfig.go

```diff
@@ -30,6 +31,7 @@ type Config struct {
 type Common struct {
 	Env       map[string]string `json:"env,omitempty"`
 	BuildArgs map[string]string `json:"buildArgs,omitempty"`
+	EnvFrom   v1.EnvFromSource  `json:"envFrom,omitempty"`
```

Comment:
I now think this should also be a list, so we won't force only a single source

Reply:
@TomerShor make sense